config/jobs/kubernetes/minikube/minikube.yaml

```diff
@@ -0,0 +1,28 @@
+presets:
+- env:
+  - name: GCS_CREDS
+    value: /root/.config/k8s-minikube-20e2437ce844.json
+  volumes:
+  - name: k8s-minikube-build-gcs
+    secret:
+      secretName: k8s-minikube-build-gcs
+      items:
+      - key: k8s-minikube-20e2437ce844.json
+        path: k8s-minikube-20e2437ce844.json
+  volumeMounts:
+  - name: k8s-minikube-build-gcs
+    mountPath: /root/.config/gcloud/k8s-minikube-20e2437ce844.json
```

Comment:
This would mount the key `k8s-minikube-20e2437ce844.json` at path `/root/.config/gcloud/k8s-minikube-20e2437ce844.json/k8s-minikube-20e2437ce844.json`
See https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod

Reply:
fixed.